.github/workflows/main.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
I'm happy to just test 18.x since 16.x is EOL.

Reply:
set most tests to 18 and dropped 16 here: https://github.com/digitalbazaar/vc/pull/161/commits/4db7ff17ad8427ba0353ce64f75aff0182de6fe9